package.json

```diff
@@ -66,6 +65,7 @@
   "dependencies": {
     "@types/react-dom": "^17.0.9",
     "graphql": "16.0.0",
+    "lerna": "^4.0.0",
```

Comment:
oops. I upgraded to Big Sur, which turned out to be a Big Pain and among other things broke storybook. In the process of fixing I found that lerna wasn't working, so reinstalled. causing this. thought to leave it in, might as well be up to date.